README.md

```diff
@@ -166,6 +167,68 @@ To do so, we are implementing anonymous diagnostic data collection.
 We will never collect any user sensitive data, only integration related
 warnings and errors. See related [developers discussion](https://github.com/leikoilja/ha-google-home/discussions/116).
 
+## Troubleshooting
+
+### Collecting useful log data
+
+Here are the steps to generate useful log data:
+
+1. Temporary log level change.
+   1. Visit [Home Assistant Developer Services Tool](https://my.home-assistant.io/redirect/developer_services/).
+   2. Choose `Logger: Set level` from the **Service** menu.
+   3. Under **Service data** paste the following:
+      ```yaml
+      custom_components.google_home: debug
+      glocaltokens: debug
+      ```
+   4. Click **Call Service**.
+2. Read the log information.
+   1. Visit [Home Assistant Logs](https://my.home-assistant.io/redirect/logs/).
+   2. Click **Load Full Home Assistant Log**.
+   3. Look for all `google_home` entries.
+   4. Also look for all `glocaltokens` entries.
+3. Requesting help with the log information.
+   1. Copy the log entries.
+   2. Paste them into a discussion forum or bug report between the following two lines:
+      > \```
+      >
+      > \```
+
+> Advanced tip for Windows users:
+>
+> Install [LogExpert](https://github.com/zarunbal/LogExpert/) to make finding log entries quicker.
+>
+> 1. Using LogExpert: Open `home-assistant.log` in your config folder.
+> 2. Use `glocaltokens|google_home` as your _Text filter_ and check `Regex` to include logs from both _glocaltokens_ and _google_home_.
+
+### "Username/Password is incorrect"
+
+If you get this error:
+
+1. First verify you are using the correct Username and Password combination for that Google account.
+2. Have you enabled 2 Factor Authentication on that Google account? _If so read the [2 Factor Authentication](#2-factor-authentication--app-passwords) section to continue._
+3. After ruling out #1 and #2: _Read the [Collecting useful log data](#collecting-useful-log-data) section._
```

Comment:
Let's add something about trying to disable other custom components which may interfere with auth process.